x-pack/test/functional/apps/ml/short_tests/embeddables/anomaly_charts_dashboard_embeddables.ts

```diff
@@ -80,13 +84,6 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         it('create new anomaly charts panel', async () => {
           await ml.dashboardEmbeddables.clickInitializerConfirmButtonEnabled();
           await ml.dashboardEmbeddables.assertDashboardPanelExists(testData.panelTitle);
-
-          await ml.dashboardEmbeddables.assertNoMatchingAnomaliesMessageExists();
```

Comment:
@pheyos removing this line did the trick as you discussed in zoom. Thanks again.